### ISSUE TITLE
Suggestions for `pixi info` and `pixi task list`  terminal ouput 

### ISSUE THREAD

**Issue by vigneshmanick** [prefix-dev/pixi#1397]
### Problem description

Hello Pixi Team,

There is an inconsistency between `pixi task list` and `pixi info` wrt. Tasks and the dependencies displayed

- `pixi task list` displays the tasks in alphabetical order while those in `pixi info` are not.
- `pixi info` shows the dependencies according to the features defined but it's not readily visible which dependency belongs to which feature. The features have different colors but the dependencies don't

#### Pixi info
![image](https://github.com/prefix-dev/pixi/assets/62500639/c6af3cac-5e18-4e0a-9526-a82d308fa4df)

#### Pixi task list
![image](https://github.com/prefix-dev/pixi/assets/62500639/11443c6c-61e1-412a-81ab-1d4a417a3ca0)

Suggestions  

- The tasks in `pixi task list ` are all bold and blue but in the `pixi info` command they are blue but not bold. The output is inline in one command and in the other is all in the same line. Suggestion here would be that `pixi task list` output be similar to pixi info (inline). When there are multiple tasks the vertical real estate is used up quickly.
- The order of tasks should be the same in both `pixi info` and `pixi task list`. imo they both should be shown sorted alphabetically and also per environment. 
- The dependencies shown in `pixi info` should have the same color of the corresponding feature and should also be sorted so that it's clear which dependency belongs to which feature.
- Maybe print the dependencies per feature?
-  The features should also be sorted alphabetically in `pixi info`. 

Tldr; Sort the data displayed while using consistent colors and formatting (`pixi info` style is well suited for different command outputs imo) across commands to differentiate/collate output.

**Comment by PanTheDev** [comment]
I'm thinking for both `pixi info` and `pixi tasks list` there should be a `--per-feature` argument to group dependencies and/or tasks by feature.

**Comment by vigneshmanick** [comment]
just tried to hack the output (with my very limited rust Kenntnisse ) a bit and have got the following for `pixi task list` and `pixi task summary`. The former outputs all the tasks inline similar to the `pixi info` command and the latter outputs the tasks per environment.


### `pixi task list`
![image](https://github.com/prefix-dev/pixi/assets/62500639/c04de2ed-f7ad-468f-8a6b-c20168c20c16)

### `pixi task list -s`
![image](https://github.com/prefix-dev/pixi/assets/62500639/3cc134d7-d46c-45e4-8ca6-15ce07ad804c)

looking at the outputs, i feel the `pixi info` command might be better suited to display these. e.g `pixi info tasks`, `pixi info tasks -s`  since all these outputs are alredy formatted and available in the `pixi info` cli class. 

Me as a user when looking at the `info` command expect that both the filtered and unfiltered info will be available in the `info` command.

**Comment by ruben-arts** [comment]
That looks and sounds logical @vigneshmanick, would you be able to make a PR with your work?
